README.md

```diff
@@ -83,6 +88,8 @@ The mentioned `parameters_file` will be created for every project with `default_
 }
 ```
 
+Project specific DAP configuration (ex. `"dap_configuration": "cppdbg"`) will also be contained here but is left out if the project uses the global default.
```

Comment:
When fixing #51, I put examples in the config above. Could you pull the master and update the instructions for consistency?

Reply:
I pulled and merged the new instructions. Did you want me to add something to it or was it just the merge?

Reply:
I meant to move `"dap_configuration"` into the config above with an example.

Reply:
I moved it. Hopefully I understood what you meant, if not then it's probably better if you write it.

Reply:
Awesome, thanks.